src/main/java/org/sagebionetworks/bridge/exporter/synapse/SynapseStatusTableHelper.java

```diff
@@ -0,0 +1,141 @@
+package org.sagebionetworks.bridge.exporter.synapse;
+
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.sagebionetworks.client.exceptions.SynapseException;
+import org.sagebionetworks.repo.model.table.ColumnModel;
+import org.sagebionetworks.repo.model.table.ColumnType;
+import org.sagebionetworks.repo.model.table.PartialRow;
+import org.sagebionetworks.repo.model.table.PartialRowSet;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import org.sagebionetworks.bridge.exporter.exceptions.BridgeExporterException;
+import org.sagebionetworks.bridge.exporter.worker.ExportTask;
+import org.sagebionetworks.bridge.exporter.worker.ExportWorkerManager;
+
+/**
+ * Highly specialized helper, which writes the status row to the Synapse status table, creating it if it doesn't
+ * already exist.
+ */
+@Component
+public class SynapseStatusTableHelper {
+    // Constants package-scoped to be available to unit tests.
+    static final String COLUMN_NAME_UPLOAD_DATE = "uploadDate";
+    static final List<ColumnModel> COLUMN_LIST;
+    static {
+        // Construct column definition list. This table is only used as a flag to signal that we're done uploading, so
+        // we only need one column with the date.
+        // NOTE: ColumnType.DATE is actually a timestamp. There is no calendar date type.
+        ColumnModel uploadDateColumn = new ColumnModel();
+        uploadDateColumn.setName(COLUMN_NAME_UPLOAD_DATE);
+        uploadDateColumn.setColumnType(ColumnType.STRING);
+        uploadDateColumn.setMaximumSize(10L);
```

Comment:
How/where is Chris going to record what data he has processed? Is it easier for end users like researchers if it's in the same status table? (Not something you need to act on, just wondering.)